curp/src/client.rs

```diff
@@ -24,7 +24,7 @@ use crate::{
     rpc::{
```

Comment:
please rebase on master and remove the `client.rs` file

Reply:
Done